cpmpy/expressions/__init__.py

```diff
@@ -18,8 +18,8 @@
 
 # we only import methods/classes that are used for modelling
 # others need to be imported by the developer explicitely
-from .variables import boolvar, intvar, cpm_array
+from .variables import boolvar, intvar, cpm_array, DirectVar
```

Comment:
for user-facing functions we are using lowercase (e.g. boolvar, intvar) I think I will lowercase this too

Reply:
or is this the _impl and should we allow shape=... too

also, there is a difference between the name of a variable and the name of the internal API call...

will think some more about it